[1,"Error occurred during cleanup: Error: The process cannot access the file because it is being used by another process. (os error 32): remove 'C:\\Users\\Phil\\AppData\\Local\\Temp\\quarto-session9e537f85e7c025fd'\n"]
